src/semantic-errors.ts

```diff
@@ -65,6 +65,7 @@ function whitelistSupportedDiagnostics(
       case 1121: // ts 3.2 "Octal literals are not allowed in strict mode."
       case 1123: // ts 3.2: "Variable declaration list cannot be empty."
       case 1141: // ts 3.2 "String literal expected."
+      case 1070: // ts 3.2 '{0}' modifier cannot appear on a type member.
```

Comment:
It's a nit, but an obvious one, please stick to the ascending order for the diagnostic code cases :D

Reply:
Makes things easier to scane

Reply:
merge messed up order xd

Reply:
fixed